### DISCUSSION TITLE
Should users be able to create metadata records with identical titles?

### DISCUSSION THREAD

**Discussion by NicoledeGreef** [bcgov/MFIN-Data-Catalogue#328] [General]
Currently, users of the Finance Data Catalogue can create more than one metadata record that has an identical title. Let's consider: does it makes sense, instead, to enforce unique titles amongst metadata records in the system?

In theory, a user could create a metadata record for a **Form** called "Property Transfer Tax Applications". In addition, they could create a metadata record for a **Data** set called "Property Transfer Tax Applications" and then metadata records for one or more **Reports** that could also be named "Property Transfer Tax Applications".

In the advent that this occurs, technically each metadata record would have a unique permalink based on the Drupal node ID (e.g. https://test.cat.data.fin.gov.bc.ca/node/99, https://test.cat.data.fin.gov.bc.ca/node/100, https://test.cat.data.fin.gov.bc.ca/node/73, etc. ) and each metadata record would specify the type of item being catalogued as well as other details that make the item distinct from others.

Interested to hear others' thoughts on this.

**Comment by CraigClark** [comment]
@NicoledeGreef 

I would leave it as is. For find-ability, the facets are designed to get around this sort of thing. In the example you give, as a user I would search for _Property Transfer Tax Applications_ and I get two results, in the facets, I see under type that there is a record and a form. 

Or, I see two, I look at both.

The system should reflect the actual datasets. So if in the real world there are two Property Transfer Tax Applications, there should be here too. 

People are people, if you force unique titles, you will see things like 'Property Transfer Tax Applications 1' 'Property Transfer Tax Applications 2' which isn't useful.
